core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/signup/OtherSignUpOptionSelectionViewModel.kt

```diff
@@ -115,13 +115,7 @@ class OtherSignUpOptionSelectionViewModel(
 
             val passkeyCreated = createPasskeyUseCase?.invoke(
                 CreatePasskeyParams(
```

Comment:
Why does the create passkey params class still need to exist? The only parameter is username.

Also : do not use .toString on a nullable string. This is just hiding when something is null that shouldn't be the case. null.toString = "null" - that is not what we want.